src/EnergyPlus/WeatherManager.cc

```diff
@@ -8077,8 +8074,6 @@ namespace Weather {
         // Create arrays (InterpolationValues, SolarInterpolationValues) dependent on
         // Number of Time Steps in Hour.  This will be used in the "SetCurrentWeather" procedure.
 
-        int halfpoint = 0;
-
```

Comment:
[src/EnergyPlus/WeatherManager.cc:8080]:(style),[unreadVariable],Variable 'halfpoint' is assigned a value that is never used.